src/components/steps/WelcomeForm.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div data-test="welcome-back" class="welcome-back">
     <div class="bar">
-      <CustomIcon class-name="row sheikah-icon" name="SheikahIcon" />
+      <CustomIcon class-name="row sheikah-icon" name="sheikah-icon" />
```

Comment:
After creating a wallet, I can't see the sync loader and it gets stuck.

### Error
![image](https://github.com/witnet/sheikah/assets/24905681/43d313bf-ca22-4e07-a48b-a4583a36f2f2)

### View
![image](https://github.com/witnet/sheikah/assets/24905681/fc9a9c82-69e9-4cd8-aa14-28aa0a97dca7)